Classes/Database/TestCase.php

```diff
@@ -467,4 +477,38 @@ protected function importDataSet($path)
             }
         }
     }
+
+    /**
+     * Populate $GLOBALS['TYPO3_DB'] and create test database
+     *
+     * @see https://github.com/Nimut/testing-framework/blob/master/src/TestingFramework/TestSystem/TestSystem.php
+     *
+     * @throws Exception
+     * @return void
+     */
+    protected function setUpTestDatabase()
```

Comment:
Why this ? is identical to the one from nimut/testing-framwork

Reply:
I can't see the connection from this extension to the nimut/testing-framework. Whats your point? ;-) I made this extension compatible to TYPO3V8 and copied a fix from nimut/testing-framework to solve the issue in this extension to setup the database settings for doctrine...

Reply:
I thought it was required.. My bad.

Would rather require as having copied code.